sphinx/ext/intersphinx.py

```diff
@@ -241,12 +243,28 @@ def load_mappings(app):
                     or inventories.cache[uri][1] < cache_time:
                 safe_inv_url = _get_safe_url(inv)  # type: ignore
                 logger.info('loading intersphinx inventory from %s...', safe_inv_url)
-                invdata = fetch_inventory(app, uri, inv)
+                try:
+                    invdata = fetch_inventory(app, uri, inv)
+                except Exception as err:
+                    failures.append(err.args)
+                    continue
+
                 if invdata:
                     inventories.cache[uri] = (name, now, invdata)
                     update = True
                     break
 
+        if len(failures) < len(invs):
+            logger.info("encountered some issues with some of the inventories,"
+                        " but they had working alternatives:")
```

Comment:
what does "alternatives" mean here? where can I set up them?

Reply:
I meant this: https://github.com/sphinx-doc/sphinx/issues/1537, usage is e.g. here:
https://github.com/astropy/astropy-helpers/blob/b5b722345db475d91edb9fbae41d3d0364d6764a/astropy_helpers/sphinx/conf.py#L51

Reply:
Thanks, I understand. I don't know such feature. I looks good.